### DISCUSSION TITLE
Crowdsourced data

### DISCUSSION THREAD

**Discussion by gpb3037** [earthlab-education/Earth-Analytics-AY24#111] [Reading Responses]
Regarding https://scitechdaily.com/revolutionizing-the-map-how-smartphones-and-crowdsourcing-are-redefining-geospatial-data/

One aspect of the crowd sourced data article that was not covered was conflation or generalization of location sources.  Most of the time, when there is a gps location detected by a phone it is not differentially corrected - that is the specific conditions of the gps signal in a specific locality are not adjusted for.  This raw GPS data is compared or snapped to or merged with other relatively close known data such as a known point entity such as named building, a linear entity such as a street to determine a route, or to a polygonal entity such as a census tract or zip code.
GPS sensors on phones or other devices are not generally intended to do high resolution location such as for surveying a property line, but to establish a relative position.  Using a street network to communicate this could be one form of geocoding to use closest address along the street in the case where there is a point source providing all the valid addresses, or to estimate the distance along the street segment as a percentage when there is not a set of valid addresses available
To communicate a spot in a field, or a place along the curb a the airport, it can be less convenient to list out a long GPS location (that in turn may be inaccurate unless corrected) thus typically a zone or other generalization is used.  One interesting one is https://what3words.com/ which divides the entire globe up into 3x3meter or 10x10ft blocks with three words uniquely defining each space (it uses shorter words in urban areas and longer words in the hinterlands) and is published in many different languages so that one could simply speak a location and get a person to a precise spot good enough for humans to use.

Earthlab is listed at 4001 Discovery Dr Suite S348 - using the campus map I can find this with 

https://www.colorado.edu/map?id=336#!ct/83261,68796,46807,68797,68838,68840,68841,68843,69772,46902,46903,46990,46991,47016,47030,47043,47044,47045,47046,47050,47054,47055,47057,47070,47071,47073,47076,47077,47078,47079,47087,47088,47090,47131,47132,47133,47134,47135,47139,47144,47149,47150,47156,47162,47163,47172,47173,47174,47175,47229,47230,47243,47247,47249,47251,47252,47253,47254,47256,47257,47258,47259,47260,47261,47262,47488,47489,47592,47593,47619?s/?mc/40.00941304468995,-105.24215968643625?z/19.035454490297727?lvl/3?share

Or I can get a good enough position with https://w3w.co/ocean.gangs.mercy and saying 3rd floor
on the base map there is a point for earth lab too at https://w3w.co/copies.dates.detect

**Comment by eculler** [comment]
Great points @gpb3037 -- we will definitely run into some of the challenges with geocoding (changing place names into coordinate values) this week using Open Street Maps. 

You can look at some neat work on using large language models (LLMs, ChatGPT being the most famous example) to geocode narrative reports in Luca Palasti's certificate project from this past year: https://github.com/lucap1211/EA_fire_project/blob/main/Blog%20posts/Final_blog_post.ipynb @lupa1211
